pkg/controllers/management/kontainerdrivermetadata/data_getter.go

```diff
@@ -16,6 +16,25 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func GetCisConfigParams(
```

Comment:
Why not just hang this off of `cisScanHandler` and keep it in the same package?

Reply:
Following the same pattern as the rest of the file.